pyanaconda/geoloc.py

```diff
@@ -704,7 +720,8 @@ def reverse_geocode_coords(self, coordinates):
             return None  # unknown geocoder specified
```

Comment:
I can't place comment near to that place but make the `Geocoder` doc class one line please.